source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/calification.js

```diff
@@ -153,6 +154,15 @@
             }
         });
 
+        this.isResposibleCalificating = ko.computed(function () {
+            if (data.UserLogged == data.Evaluation.ResponsibleId) {
+                return !_.some(self.califications, function (califications) {
+                    return califications.owner == 1 && califications.finished
+                });
+            }
+            return false;
```

Comment:
It seems that `isResposibleCalificating` does not have _observable_ dependencies, is it right?

So, if I am not wrong, probably it is not necessary to implement it using _computed_ nor _observable_, maybe a _boolean_  property is enough.